docs-devsite/vertexai-preview.md

```diff
@@ -25,8 +25,15 @@ The Vertex AI in Firebase Web SDK.
 
 |  Class | Description |
 |  --- | --- |
+|  [ArraySchema](./vertexai-preview.arrayschema.md#arrayschema_class) | Schema class for "array" types. The <code>items</code> param should refer to the type of item that can be a member of the array. |
+|  [BooleanSchema](./vertexai-preview.booleanschema.md#booleanschema_class) | Schema class for "boolean" types. |
 |  [ChatSession](./vertexai-preview.chatsession.md#chatsession_class) | ChatSession class that enables sending chat messages and stores history of sent and received messages so far. |
 |  [GenerativeModel](./vertexai-preview.generativemodel.md#generativemodel_class) | Class for generative model APIs. |
+|  [IntegerSchema](./vertexai-preview.integerschema.md#integerschema_class) | Schema class for "integer" types. |
+|  [NumberSchema](./vertexai-preview.numberschema.md#numberschema_class) | Schema class for "number" types. |
+|  [ObjectSchema](./vertexai-preview.objectschema.md#objectschema_class) | Schema class for "object" types. The <code>properties</code> param must be a map of Schema. |
+|  [Schema](./vertexai-preview.schema.md#schema_class) | Parent class encompassing all Schema types, with static methods that allow building specific Schema types. This class can be converted with JSON.stringify() into a JSON string accepted by Vertex REST endpoints. (This string conversion is automatically done when calling SDK methods.) |
```

Comment:
NOT BLOCKING

JSON.stringify() should be tagged as code

(comment applies to several other instances, even in other files)

Reply:
Fixed.